extendr-api/src/robj/from_robj.rs

```diff
@@ -126,7 +126,7 @@ impl<'a> FromRobj<'a> for Vec<String> {
         } else if let Some(v) = robj.as_string_vector() {
             let str_vec = v.to_vec();
             // check for NA's in the string vector
-            if let Some(_str) = str_vec.iter().find(|&s| *s == na_str()) {
+            if let Some(_str) = str_vec.iter().find(|&s| *s == <&str>::na()) {
```

Comment:
Should this use `is.na()`? We should generally avoid comparison to `NA` values, just like in R.

Reply:
I checked it and now I am not sure we can replace this. It is an old implementation, and `&s` here is `&String`.
It seems there are no tests to verify that if I change it to something like `(&s[..]).is_na()`, it will still work correctly.
I suggest keeping it 'as-is' and later deprecate these conversions.

Reply:
I mean, in this case it seems that we search for literal `"NA"` in the string vector, so `<&str>::is_na()` will not work.

Reply:
Thanks for checking. Could you maybe add a brief comment describing what is happening here?